client/statistics/js/searchstats.js

```diff
@@ -1,19 +1,19 @@
 import '../sass/statistics.sass'
-import ReactModal from "react-modal";
+import ReactModal from 'react-modal'
 
-const React = window.React
-const Modifier = window.DraftJS.Modifier
-const EditorState = window.DraftJS.EditorState
-const TooltipEntity = window.draftail.TooltipEntity
-const Icon = window.wagtail.components.Icon
+const { React } = window
+const { Modifier } = window.DraftJS
+const { EditorState } = window.DraftJS
+const { TooltipEntity } = window.draftail
+const { Icon } = window.wagtail.components
```

Comment:
I'm intrigued by what the rule is that creates this structure. Also noting that we could even shorten one of these lines to

```js
const { EditorState, Modifier } = window.DraftJS
```

which is more parallel with how we usually do imports. Just an optional nitpick though.

Reply:
It was being caused by https://eslint.org/docs/rules/prefer-destructuring. But since I think we will be using destructuring probably quite a lot in our code, I have disabled that in eslint for now, and reverted this code changes.